wgsl/index.bs

```diff
@@ -9498,13 +9762,14 @@ The remaining subsections specify a static analysis that verifies that
 control flow=].
 
 <div class="note">Note: This analysis has the following desirable properties:
-      - Sound (meaning that it rejects every program that would break the uniformity requirements of builtins)
-      - Linear time complexity (in the number of tokens in the program)
-      - Refactoring a piece of code into a function, or inlining a function, cannot make a shader invalid if it was valid before the transformation
-      - If the analysis refuses a program, it provides a straightforward chain of implications that can be used by the user agent to craft a good error message
+      - Sound, meaning that a [=uniformity failure=] [=behavioral requirement|will=] be triggered for a program that would break the uniformity requirements of builtins.
+      - Linear time complexity, in the number of tokens in the program.
+      - Refactoring a piece of code into a function, or inlining a function, cannot make a shader invalid if it was valid before the transformation.
+      - If the analysis refuses a program, it provides a straightforward chain of implications that can be used by the user agent to craft a good error message.
 </div>
 
-Each function is analyzed, verifying that there is a context where it is safe to call this function. It rejects the program as invalid if there is no such context.
+Each function is analyzed, verifying that there is a context where it is safe to call this function.
+It triggers a [=uniformity failure=] if there is no such context.
```

Comment:
Nit: This sounds like we are trying to determine if there exists "a context" where it is safe to call the function. We are actually trying to ensure that _all_ contexts where the function is called are safe.

Reply:
Rewrote as:

> Each function is analyzed, trying to ensure two things:

> * Uniformity requirements are satisfied when it calls other functions, and

> * Uniformity requirements are satisfied whenever it is called.

> A [uniformity failure](https://github.com/gpuweb/gpuweb/pull/3713#uniformity-failure) is triggered if either of these two checks fail.


> As part of this work, the analysis computes metadata...